<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IpapConsole</class>
 <widget class="QDialog" name="IpapConsole">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>477</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>101</red>
        <green>148</green>
        <blue>235</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Highlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>101</red>
        <green>148</green>
        <blue>235</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>101</red>
        <green>148</green>
        <blue>235</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Highlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>101</red>
        <green>148</green>
        <blue>235</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>101</red>
        <green>148</green>
        <blue>235</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Highlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>127</red>
        <green>125</green>
        <blue>123</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Icepap Console</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icepapcms.qrc">
    <normaloff>:/icons/icons/gnome-terminal.png</normaloff>:/icons/icons/gnome-terminal.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Icepap host</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="txtHost"/>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="btnDisconnect">
     <property name="text">
      <string>Disconnect</string>
     </property>
     <property name="icon">
      <iconset resource="icepapcms.qrc">
       <normaloff>:/icons/icons/disconnect.png</normaloff>:/icons/icons/disconnect.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="btnConnect">
     <property name="text">
      <string>Connect</string>
     </property>
     <property name="icon">
      <iconset resource="icepapcms.qrc">
       <normaloff>:/icons/icons/connect.png</normaloff>:/icons/icons/connect.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="5">
    <widget class="PyConsoleText" name="console">
     <property name="cursor" stdset="0">
      <cursorShape>IBeamCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="undoRedoEnabled">
      <bool>false</bool>
     </property>
     <property name="lineWrapMode">
      <enum>QTextEdit::WidgetWidth</enum>
     </property>
     <property name="acceptRichText">
      <bool>false</bool>
     </property>
     <property name="cursorWidth">
      <number>5</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PyConsoleText</class>
   <extends>QTextEdit</extends>
   <header>.pyconsoletext</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>txtHost</tabstop>
  <tabstop>btnConnect</tabstop>
  <tabstop>btnDisconnect</tabstop>
  <tabstop>console</tabstop>
 </tabstops>
 <resources>
  <include location="icepapcms.qrc"/>
 </resources>
 <connections/>
</ui>
